### PR TITLE
PR for readme file.

### PR DESCRIPTION
**Pull request by adidoshi**
### Hey I created README file for e-commerce project.
* Fix links for navbar.
* Media queries update.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -159,6 +172,16 @@
...
 /* media queries */
 
+@media screen and (max-width: 1250px) {
+  .nav-center input {
+    display: none;
+  }
+
+  .grid-col-3 {
+    grid-template-columns: 1fr 1fr;
+  }
```
Comment (himadri2110) [inline style.css:182]: Suggestion: Classname is `grid-col-3`, so the property should also specify 3 columns of 1fr each.

**Review by himadri2110 — APPROVED** [review]
Clean UI, good to go!

File: style.css (hunk 1 above)
Comment (adidoshi) [reply style.css:182]: Thanks for this. Actually for media query it changes. Noting this down for further refactoring.

**Review by KaustubhM1997 — APPROVED** [review]
Looks great!

**Pull request merged**